### ISSUE TITLE
Calls from HP-Admin UI to HPOS over websocket timeout

### ISSUE THREAD

**Issue by peeech** [Holo-Host/holo-nixpkgs#324]
**Description**
Web socket connection to holochain instances responds with `Err.Internal.Kind: Timeout`. On current combo of frontend and backend I am getting the following errors:
1. `👎 hha/host/get_all_apps zome call ERROR using args:  {}`:
```
- req: {"jsonrpc":"2.0","method":"call","params":{"instance_id":"holo-hosting-app","zome":"host","function":"get_all_apps","args":{}},"id":1}
- res: {"jsonrpc":"2.0","result":"{\"Err\":{\"Internal\":\"{\\\"kind\\\":\\\"Timeout\\\",\\\"file\\\":\\\"crates/core/src/nucleus/ribosome/runtime.rs\\\",\\\"line\\\":\\\"220\\\"}\"}}","id":1}
```
2. `👎 hha/host/get_enabled_app_list zome call ERROR using args:  {}`
```
- req: {"jsonrpc":"2.0","method":"call","params":{"instance_id":"holo-hosting-app","zome":"host","function":"get_enabled_app_list","args":{}},"id":2}
- res: {"jsonrpc":"2.0","result":"{\"Err\":{\"Internal\":\"{\\\"kind\\\":\\\"Timeout\\\",\\\"file\\\":\\\"crates/core/src/nucleus/ribosome/runtime.rs\\\",\\\"line\\\":\\\"220\\\"}\"}}","id":2}
```
3. `👎 holofuel/transactions/list_pending zome call ERROR using args:  {}`
```
- req: {"jsonrpc":"2.0","method":"call","params":{"instance_id":"holofuel","zome":"transactions","function":"list_pending","args":{}},"id":3}
- res: {"jsonrpc":"2.0","result":"{\"Err\":{\"Internal\":\"{\\\"kind\\\":\\\"Timeout\\\",\\\"file\\\":\\\"crates/core/src/nucleus/ribosome/runtime.rs\\\",\\\"line\\\":\\\"220\\\"}\"}}","id":3}
```

**To Reproduce**
1. Log in into your HP Admin UI
2. Observe console and network traffic

**Expected behavior**
Zome calls should return `{"jsonrpc":"2.0","result":"{\"Ok\" ...`

**Additional context**
My HPOS is running on `holo-nixpkgs` branch `2020-01-22-rc-1-internal` current head at `ef6a020879b46a9ce1360472fde638d895d9b959` with is pointing to `hp-admin` branch `release-0.10.0` current head at `2203eda5d68164645a5b044bcc68d769fc150ebf`.

**Comment by zo-el** [comment]
This is fixed by updating the holochain to `v0.0.42-alpha5`

**Comment by peeech** [comment]
Resolved in #325 and #326

**Comment by yegortimoshenko** [comment]
#325 has not been merged, so AFAIK this isn't resolved in `develop` yet?
